src/output-json-stats.c

```diff
@@ -282,6 +297,10 @@ json_t *StatsToJSON(const StatsTable *st, uint8_t flags)
                 }
 
                 if (js_type != NULL) {
+                    if (is_exception_policy_counter && st->stats[u].value == 0 &&
+                            !stats_eps_zero_counters) {
+                        continue;
+                    }
```

Comment:
This is part of what causes the address sanitizer error. Removing the wrong portion of the stats, most likely.